SimCalorimetry/HGCalSimProducers/interface/HGCDigitizerBase.h

```diff
@@ -131,8 +137,13 @@ class HGCDigitizerBase {
   //front-end electronics model
   std::unique_ptr<HGCFEElectronics<DFr> > myFEelectronics_;
 
+  static const int ss = (int)hgc::nSamples;
```

Comment:
'ss' is not a very good variable name, please make it more descriptive (and also include the trailing underscore)

Reply:
Alternatively, you could just use the constexpr `hgc::nSamples` directly for the array size below